app/animals/cat.py

```diff
@@ -0,0 +1,11 @@
+from app.animals.animal import Animal
+
+
+class Cat(Animal):
+
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=3, is_hungry=is_hungry)
+
```

Comment:
no need to pass required parameters explicitly. we often do it when we're gonna pass not required parameter and there are many of them like
```
def func(param1, param2, param3, unrequired1=None, unrequired2=None, unrequired3=None, unrequired4=None, unrequired5=None, unrequired6=None):
    ...

# call function with explicit unrequired6 param so that Python knows that "Ihor" isn't a value for unrequired1
func(1, 2, 3, unrequired6="Ihor")
# instead of this
func(1, 2, 3, None, None, None, None, None, "Ihor")
```
Fix everywhere